src/librustc/session/config.rs

```diff
@@ -1285,6 +1283,85 @@ pub fn parse_crate_types_from_list(list_list: Vec<String>) -> Result<Vec<CrateTy
     return Ok(crate_types);
 }
 
+pub mod nightly_options {
+    use getopts;
+    use syntax::feature_gate::UnstableFeatures;
+    use super::{ErrorOutputType, OptionStability, RustcOptGroup, get_unstable_features_setting};
+    use session::{early_error, early_warn};
+
+    pub fn is_unstable(option_name: &str, flags: &[RustcOptGroup]) -> bool {
+        flags.iter().any(|x| {
+            x.opt_group.long_name == option_name || x.opt_group.short_name == option_name
+        })
+    }
```

Comment:
Bad code legacy. Initially, it tested if an option was unstable, but it doesn't anymore.